is_prime/program/src/main.rs

```diff
@@ -0,0 +1,33 @@
+//! A simple program that takes a number `n` as input, and writes the `n-1`th and `n`th fibonacci
+//! number as an output.
+
+// These two lines are necessary for the program to properly compile.
+//
+// Under the hood, we wrap your main function with some extra code so that it behaves properly
+// inside the zkVM.
+#![no_main]
+sp1_zkvm::entrypoint!(main);
+
+use alloy_sol_types::SolType;
+use fibonacci_lib::{is_prime, PublicValuesStruct};
+
+pub fn main() {
+    // Read an input to the program.
+    //
+    // Behind the scenes, this compiles down to a custom system call which handles reading inputs
+    // from the prover.
+    let n = sp1_zkvm::io::read::<u32>();
+
+    // Compute the n'th fibonacci number using a function from the workspace lib crate.
```

Comment:
This comment and the next line seem rather confusing.

Are we doing primes or Fibonacci?